### ISSUE TITLE
DivMarker / DivIcon classname

### ISSUE THREAD

**Issue by zach-morris** [thedirtyfew/dash-leaflet#95]
I'm attempting to use DivMarkers, the current class defines the iconoptions as follows:

```
iconOptions: PropTypes.shape({
        iconSize: PropTypes.arrayOf(PropTypes.number),
        iconAnchor: PropTypes.arrayOf(PropTypes.number),
        popupAnchor: PropTypes.arrayOf(PropTypes.number),
        className: PropTypes.arrayOf(PropTypes.number),
        html: PropTypes.string
    }),
```

I believe this should be:
```
iconOptions: PropTypes.shape({
        iconSize: PropTypes.arrayOf(PropTypes.number),
        iconAnchor: PropTypes.arrayOf(PropTypes.number),
        popupAnchor: PropTypes.arrayOf(PropTypes.number),
        className: PropTypes.string,
        html: PropTypes.string
    }),
```

**Comment by emilhe** [comment]
Ah, yes. I have changed className to string. It should make it into the next release.

**Comment by emilhe** [comment]
The bug should be fixed in `dash-leaflet==0.1.16`.
